lib/Config.js

```diff
@@ -243,6 +243,9 @@ class Config {
         if (process.env.S3KMS) {
             kms = process.env.S3KMS;
         }
+        if (process.env.S3DP) {
+            data = process.env.S3DP;
+        }
```

Comment:
I think it would be clearer if you had a separate variable called db or erasure coding and set that separately from data.  Then that variable would be set to on or off.